print(11/0)
#Error will come.